assemble/src/main/scripts/generate-download-script.sh

```diff
@@ -0,0 +1,56 @@
+#! /usr/bin/env bash
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script will generate a DEPENDENCIES listing of packaged dependencies
+
+in=target/dependencies.raw.txt
+out=target/download-dependencies
+
+cat >"$out" <<'EOF'
+#! /usr/bin/env bash
+# This script downloads the following jars, identified by their maven
+# coordinates, using the maven-dependency-plugin.
+#
+# DISCLAIMER: This is only one possible way to download a set of dependencies
```

Comment:
> Because we're not developing those projects, and there is a separation/delegation of responsibilities. We are not the upstream for those projects. We do not control the direction, compatibility, or bugfixes of those dependencies, or their release cycles. They do not fall under the Accumulo PMC responsibilities, and if we tried to take that on, I think it would be too much for our small community to cover, and it would introduce scope creep into the Accumulo project.
> I think it's much healthier for us to engage and develop those connections with those other projects, and users, rather than try to take on a scope which is inherently their responsibility.

Long term, perhaps. But this seems like a long path.

> Do you mean relocate, like shading does? Because I'm definitely not a fan of that. That can cause a lot of hidden problems (can't relocate dynamically loaded classes, for one), and would add a lot of extra work for downstream packagers who are trying to do dependency convergence/integration under a no-bundling/dynamic-libs-only environment like most Linux distributions.

It doesn't invalidate the ability for downstream people to still do whatever they want. These classes should also not be getting dynamically loaded, should they? They would comprise the "core" of Accumulo. It would simplify the case for users who just want to get up and running (that are not release engineering experts).